### ISSUE TITLE
[REG] GroveLCD doesn't work on Arduino

### ISSUE THREAD

**Issue by qiaojingx** [01org/zephyr.js#723]
Flashing GroveLCD related samples([GroveLCD.js](https://github.com/01org/zephyr.js/blob/master/samples/GroveLCD.js), [WebBluetoothGroveLcdDemo.js](https://github.com/01org/zephyr.js/blob/master/samples/WebBluetoothGroveLcdDemo.js), [HeartRateDemo.js](https://github.com/01org/zephyr.js/blob/master/samples/HeartRateDemo.js)) and tests([test-grovelcd.js](https://github.com/01org/zephyr.js/blob/master/tests/test-grovelcd.js), [test-grovelcd-cursor-manual.js](https://github.com/01org/zephyr.js/blob/master/tests/test-grovelcd-cursor-manual.js),[test-grovelcd-ds-manual.js](https://github.com/01org/zephyr.js/blob/master/tests/test-grovelcd-ds-manual.js), [test-grovelcd-rgb-manual.js)](https://github.com/01org/zephyr.js/blob/master/tests/test-grovelcd-rgb-manual.js) on Arduino 101 with commit 3f8eb85. Fail to start LCD due to `[ERROR] JerryScript: cannot run javascript`

Serial logs:
```
ipm_console0: 'Sensor core running ZJS ARC support image'
src/zjs_grove_lcd_ipm.c:62 zjs_glcd_call_remote_function():
[ERROR] error code: 3
src/main.c:226 main():
[ERROR] JerryScript: cannot run javascript
```
Branch | Target Device | Build ID | Date | Result
---|---|---|---|---
master | Arduino 101 | 21915c0 | Feb, 2017 | PASS
master | FRDM-K64F | 3f8eb85 | Feb 13, 2017 | PASS
master | Arduino 101 | 3f8eb85 | Feb 13,2017 | FAIL

**Comment by qiaojingx** [comment]
This issue will be fixed by #722

**Comment by qiaojingx** [comment]
This issue is not reproduced on Arduino 101 and FRDM-K64F with commit 7c5ff3c.
Closed
